src/client/game/conundrum-round/ddoris.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+
+import style from './conundrum-round.scss';
+
+import ddoris from './ddoris.jpg';
+
+let dDorisTimer;
+export default class DDoris extends Component {
+    componentWillMount() {
+        dDorisTimer = setTimeout(this.fadeDDoris.bind(this), 1000);
```

Comment:
She likes it when I call her dolly, makes her feel young. Reminds her of the nights out in the dance halls bopping along to Billy Ternent and Jazz boys. Getting flirted on by all those lovely american soldiers before they go off to war. 

:+1: on the timing question